spring-integration-core/src/main/java/org/springframework/integration/channel/AbstractSubscribableChannel.java

```diff
@@ -37,19 +38,30 @@
  * @author Gary Russell
  */
 public abstract class AbstractSubscribableChannel extends AbstractMessageChannel
-		implements SubscribableChannel {
+		implements SubscribableChannel, SubscribableChannelManagement {
 
 	private final AtomicInteger handlerCounter = new AtomicInteger();
 
+	@Override
+	public int getSubscriberCount() {
+		MessageDispatcher dispatcher = getRequiredDispatcher();
+		if (dispatcher instanceof AbstractDispatcher) {
+			return ((AbstractDispatcher) dispatcher).getHandlerCount();
+		}
+		return this.handlerCounter.get();
```

Comment:
Great! 
But I think we can't backport it to `4.3.x`. 
Therefore that work should be done in the separate PR.

What is in your mind ?